assets/js/signup.js

```diff
@@ -23,7 +11,7 @@ var loadQuestions = function() {
   });
 
   $.getJSON({
-    url: '/server/form.php?retrieve=questions',
+    url: '/server/form.php?retrieve=questions&subheadings[]=Contact Information&subheadings[]=Language Information&subheadings[]=Background Information',
```

Comment:
I think a config file might be too fragile because how will you specify which question goes in which order? We can't do it by an id since those are subject to change.. I guess we could do it by the tag, but that seems pretty wonky to me. "Check this file every time we load the appointment sign up page, and sort by whatever this file says." <- That seems janky, it would be far easier to specify it in SQL and do a simple ORDER BY index ASC.